templates/ci-linux.template.yml

```diff
@@ -1,73 +0,0 @@
-# This file is auto-generated
-#
-# To regenerate it run
-#
-#     make github-actions
-#
-name: Linux
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-  release:
-    types:
-      - created
-
-jobs:
-{% for job in jobs %}
```

Comment:
i wonder why a matrix was not used in the first place, were they not available since the beginning of gha?

Reply:
No idea, I believe they were available.